src/client/app/heroes/heroes/heroes.component.ts

```diff
@@ -76,7 +75,7 @@ export class HeroSearchComponent implements OnDestroy, OnInit {
   }
 
   getHeroes() {
-    this.heroDispatcher.getAll();
+    this.heroService.getAll();
     this.unselect();
```

Comment:
we unselect prior to the delete, but after we do a `get`. Should we be consistent? I dont think order matters here.